src/manager/components/AddMealForm/AddMealForm.tsx

```diff
@@ -0,0 +1,84 @@
+import { yupResolver } from '@hookform/resolvers/yup';
```

Comment:
I like the way you organised the files. Should work really well with the way the app is designed but let's make sure it stays consistent with the rest of the code once you merge.